Makefile

```diff
@@ -53,4 +53,8 @@ clean: ## Cleans all unversioned files in project
 
 .PHONY: debug-github-workflow
 debug-github-workflow: ## Runs the github worflow locally act is needed on the system
-	@act -v -s DOCKER_HUB_USER=docke_hub_username -s DOCKER_HUB_PASSWORD=docke_hub_password
\ No newline at end of file
+	@act -v \
+		-s DOCKER_HUB_USER=dockerhub_username \
```

Comment:
this ``dockerhub_username`` is not  ``$dockerhub_username``? Where is defined?

Reply:
these are just examples. remaining them.

Reply:
examples in the makefile? ok... i find it odd :-)

Reply:
To properly state it, they are placeholders fro your credentials. I could over engineer it by adding a `.env` file where you are supposed to write these as environment variables and they are local for you machine only.

Reply:
Added `.env` file and fixed `Makefile` + `README.md` in 8a9c910
It will result much cleaner now.